interface/usergroup/usergroup_admin.php

```diff
@@ -184,9 +186,12 @@
                     );
                 }
                 $olduf["$facid/$whid"] = false;
+                /******************************************************
+                // What's the intent here? $deffacid does not appear anywhere else in the code.
                 if ($facid == $deffacid) {
                     $deffacid = 0;
                 }
+                ******************************************************/
```

Comment:
vote to remove it :)

![dwight-schrute-the-office](https://user-images.githubusercontent.com/278968/129305503-4247c3ab-916d-437e-bd4d-3c7e32a497f2.gif)